lib/utils/import/evernote/index.js

```diff
@@ -0,0 +1,158 @@
+const fs = __non_webpack_require__('fs'); // eslint-disable-line no-undef
```

Comment:
I realized we're going to need to do some conditional loading here so it doesn't break in browsers (or jest).

↓ Something like we're doing in app.jsx

```
let fs = null;
const foundElectron = has(window, 'process.type');

if (foundElectron) {
  fs = __non_webpack_require__('fs'); // eslint-disable-line no-undef
}
```